### PR TITLE
Use Fade animation when showing/hiding Tooltips

### PR DESCRIPTION
**Pull request by csillag**
As requested here: https://app.clickup.com/t/862j7t8az

### CODE REVIEW

**Comment by csillag** [comment]
Btw, here is the list of supported animations: https://mui.com/material-ui/react-tooltip/#transitions

And feedback from @donouwens : 
> Yeah Fade is the right one

File: src/app/components/CopyToClipboard/index.tsx [hunk 1]
```diff
@@ -47,7 +48,15 @@ export const CopyToClipboard: FC<CopyToClipboardProps> = ({ label, value }) => {
   }, [])
 
   return (
-    <Tooltip arrow onOpen={hideTooltip} open={isCopied} placement="right" title={t('clipboard.success')}>
```
Comment (lukaw3d) [inline src/app/components/CopyToClipboard/index.tsx:50]: Title is inaccurate - we already had a default animation

| Before | After |
| --- | --- |
|  Recording 2023-02-27 at 19 24 26 | r |

Comment (lukaw3d) [reply src/app/components/CopyToClipboard/index.tsx:50]: And this is now inconsistent with other tooltips

Recording 2023-02-27 at 19 48 43

Comment (csillag) [reply src/app/components/CopyToClipboard/index.tsx:50]: Indeed, so the default animation was Grow, which we are replacing with Fade now, as specified by @donouwens.
I have updated the title of the PR to reflect that.
Indeed, there are other places where we also have similar tooltips, I am guessing that we should replace all of them, too, but I have asked @donouwens to confirm before proceeding.

Comment (csillag) [reply src/app/components/CopyToClipboard/index.tsx:50]: I have already updated the code to use `Fade` everywhere. 
And according to @donouwens :

> My preference is Fade - as it’s a bit calmer.

Comment (csillag) [reply src/app/components/CopyToClipboard/index.tsx:50]: > I have already updated the code to use Fade everywhere.

I mean, all the _two_ locations.

Comment (lukaw3d) [reply src/app/components/CopyToClipboard/index.tsx:50]: Timings aren't consistent :D 

perhaps defaultTheme is a better place for this
```ts
    MuiTooltip: {
      defaultProps: {
        TransitionComponent: Fade,
        TransitionProps: { timeout: 600 },
      },
```

**Review by lukaw3d — APPROVED** [review]

File: src/app/components/CopyToClipboard/index.tsx (hunk 1 above)
Comment (csillag) [reply src/app/components/CopyToClipboard/index.tsx]: Indeed. Moved in there.

**Pull request merged**